Skill/app/src/main/java/com/skilltradiez/skilltraderz/EditSkillActivity.java

```diff
@@ -22,92 +22,97 @@
 import android.app.AlertDialog;
 import android.content.Context;
 import android.content.DialogInterface;
+import android.content.Intent;
+import android.graphics.Bitmap;
 import android.os.Bundle;
 import android.os.StrictMode;
+import android.provider.MediaStore;
 import android.view.View;
 import android.widget.ArrayAdapter;
 import android.widget.Button;
 import android.widget.CheckBox;
 import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
 import android.widget.Spinner;
 import android.widget.Toast;
 
-/**~~TYPE: MODEL + CONTROLLER
```

Comment:
Not sure what happened with the comment formatting here. I think it was the Ctrl-Alt-L reformatter actually! So maybe we shouldn't use that.